js/jquery.indiciaMapPanel.js

```diff
@@ -2624,7 +2624,7 @@ var destroyAllFeatures;
           layerTitle,
           div.settings.indiciaGeoSvc + 'wms',
           { layers: value, transparent: true },
-          { singleTile: true, isBaseLayer: false, sphericalMercator: true }
+          { singleTile: true, isBaseLayer: false, sphericalMercator: true, isIndiciaWMSLayer: true }
```

Comment:
This seemed like the best way to flag an Indicia WMS layer - it makes for the most readable code later.